extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ArcProcessor.java

```diff
@@ -281,6 +296,10 @@ public void transform(TransformationContext transformationContext) {
         for (QualifierRegistrarBuildItem registrar : qualifierRegistrars) {
             builder.addQualifierRegistrar(registrar.getQualifierRegistrar());
         }
+        // register additional stereotypes
+        for (StereotypeRegistrarBuildItem registrar : stereotypeRegistrars) {
+            builder.addStereotypeRegistrar(registrar.getStereotypeRegistrar());
```

Comment:
Shouldn't we also deprecate `io.quarkus.arc.processor.BeanProcessor.Builder.setAdditionalStereotypes()`?

Reply:
This PR already does that: https://github.com/quarkusio/quarkus/pull/26264/files#diff-c9b2b8f7f82a42a3270b51243922c82ca8e6617e554a67788a60ac8381701391R545